client/js/lounge.js

```diff
@@ -75,6 +78,80 @@ $(function() {
 
 	var favicon = $("#favicon");
 
+	// Autocompletion Strategies
+
+	var emojiStrategy = {
+		id: "emoji",
+		match: /\B:([-+\w]*)$/,
+		search: function(term, callback) {
+			callback($.map(emojiKeywords, function(e) {
+				return e.indexOf(term) === 1 ? e : null;
+			}));
+		},
+		template: function(value) {
+			return emojiList[emojiKeywords.indexOf(value)] + " " + value;
```

Comment:
Here's a suggestion:
```js
return `<span class="emoji">${emojiList[emojiKeywords.indexOf(value)]}</span> ${value}`;
```
```css
.textcomplete-item .emoji {
	width: 16px;
	text-align: center;
	display: inline-block;
}
```

This would introduce a nice consistency, since not all emojis are exactly sized.